### PR TITLE
Preliminary Translation of Tutorial/part zero

### PR DESCRIPTION
**Pull request by mkhan037**
Initial translation of Tutorial/part zero. Let me know whether this needs further edits and polish.

### CODE REVIEW

File: docs/tutorial/part-zero/index.md [hunk 1]
```diff
@@ -1,325 +1,325 @@
...
+একটু সময় নিয়ে কমান্ড লাইন ইন্টারফেসটি (CLI) খুঁজে নিন এবং চালু করুন। কোন অপারেটিং সিস্টেম ব্যবহার করছেন তার উপর নির্ভর করে দেখুন [**Mac এর জন্য নির্দেশাবলী**](http://www.macworld.co.uk/feature/mac-software/how-use-terminal-on-mac-3608274/), [**Windows এর জন্য নির্দেশাবলী**](https://www.lifewire.com/how-to-open-command-prompt-2618089) অথবা [**Linux এর জন্য নির্দেশাবলী**](https://www.howtogeek.com/140679/beginner-geek-how-to-start-using-the-linux-terminal/)
 
-_**Note:** If you’re new to the command line, "running" a command, means "writing a given set of instructions in your command prompt, and hitting the Enter key". Commands will be shown in a highlighted box, something like `node --version`, but not every highlighted box is a command! If something is a command it will be mentioned as something you have to run/execute._
+_**বিঃদ্রঃ:** আপনি যদি কমান্ড লাইন ব্যবহারে নতুন হয়ে থাকেন, কমান্ড রান করানোর অর্থ হচ্ছে কমান্ড প্রম্পট এ কিছু প্রদত্ত  ইন্সট্রাকশন  লিখে তা  "এন্টার" দেয়া। কমান্ড গুলো হাইলাইটেড বক্সে দেখানো হবে, যেমন `node --version`, কিন্তু সকল হাইলাইটেড বক্স ই কমান্ড নয় ! কোন কিছু যদি কমান্ড হয়ে থাকে তাহলে সেটা আপনাকে চালাতে অথবা এক্সিকিউট করতে বলা হবে।_
```
Comment (shakib609) [inline docs/tutorial/part-zero/index.md]:
```suggestion
_**বিঃদ্রঃ:** আপনি যদি কমান্ড লাইন ব্যবহারে নতুন হয়ে থাকেন, কমান্ড রান করানোর অর্থ হচ্ছে কমান্ড প্রম্পট এ কিছু প্রদত্ত ইন্সট্রাকশন লিখে তা "এন্টার" দেয়া। কমান্ড গুলো হাইলাইটেড বক্সে দেখানো হবে, যেমন `node --version`, কিন্তু সকল হাইলাইটেড বক্স ই কমান্ড নয় ! কোন কিছু যদি কমান্ড হয়ে থাকে তাহলে সেটা আপনাকে চালাতে অথবা এক্সিকিউট করতে বলা হবে।_
```

Comment (mkhan037) [reply docs/tutorial/part-zero/index.md]: Added the suggested changes along with some other whitespace correction in a new commit.

Comment (shakib609) [reply docs/tutorial/part-zero/index.md]: Thanks. Will review it fully as soon as I get some time. :)

**Review by nutboltu — APPROVED** [review]
LGTM. @shakib609 if you get time would you take a final look?

**Review by shakib609 — APPROVED** [review]
Looks good  to me too. Great work @mkhan037 :v:

**Pull request merged**